tdex-daemon.md

```diff
@@ -116,6 +116,21 @@ $ tdex --help
 
 Now you are ready to [deposit funds](#deposit-funds) to create your first market and start accepting incoming trades.
 
+## NEW: Run with Elements node as explorer
+
+It is now possible to connect the daemon directly to an Elements node instead of sourcing blockchain data from the default electrs explorer.
+You can use the `TDEX_ELEMENTS_RPC_ENDPOINT` env var to set the endpoint for connecting the daemon to the node. Currently, only insecure connection (no TLS encryption) is available.
```

Comment:
Mention also the special conf option that is needed `rpcworkqueue`, `txindex` (I guess), `server=1` and wallet must be enabled.

Reply:
Link to tdex-box compose for reference